src/components/CheckoutDialog.jsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogClose,
+  DialogTrigger,
+  DialogContent,
+  Button,
+  Flex,
+} from "@sparrowengg/twigs-react";
+import ShippingAddress from "./ShippingAddress";
+import OrderSummary from "./OrderSummary";
+export const CheckoutDialog = () => {
+  return (
+    <>
+      <Dialog>
+        <DialogTrigger asChild>
+          <Button size="sm" css={{ padding: "$10", marginBottom: "$5" }}>
+            Click to Proceed
+          </Button>
+        </DialogTrigger>
+        <DialogContent>
+          <DialogTitle></DialogTitle>
+          <ShippingAddress />
+          <OrderSummary />
+
+          <Flex>
```

Comment:
@arun-121 is this flex component needed???

Reply:
removed unnecessary wrapping